js-envs/src/main/scala/org/scalajs/jsenv/nodejs/NodeJSEnv.scala

```diff
@@ -48,11 +47,11 @@ class NodeJSEnv private (
   lazy val hasSourceMapSupport: Boolean = {
     val code = new MemVirtualJSFile("source-map-support-probe.js")
       .withContent("""require('source-map-support').install();""")
-    val runner =
-      jsRunner(CompleteClasspath.empty, code, NullLogger, NullJSConsole)
+    val sjsCode = new MemVirtualJSFile("dummy-sjs.js")
+    val runner = jsRunner(Seq.empty, sjsCode, code)
```

Comment:
With a `Seq[VirtualJSFile]`, this code wouldn't have to come up with a dummy `sjsCode`.